contractcourt/channel_arbitrator.go

```diff
@@ -30,6 +30,12 @@ var (
 		"process of being force closed")
 )
 
+const (
+	// anchorSweepConfTarget is the conf target used when sweeping
+	// commitment anchors.
+	anchorSweepConfTarget = 6
```

Comment:
Drive by comment, but perhaps this shouldn't be a constant, and instead should be some factor of the CLTV delay with any incoming HTLCs (if they exist), otherwise falling back to a distinct value.

Reply:
I'd be worried that if we have plenty of blocks left, we would pick a high conf target. Then if we get closer to the deadline, we don't change our conf target and might miss it. I think it is safer to always use `6` blocks until we have a sweeper that is aware of the deadline. Also the signed commit tx is estimated with a `3` block target, so fees need to go up quite a bit first before the `6` block target sweep can cpfp. (Although I sometimes see that bitcoind `estimatesmartfee` gives me the same estimate for all targets <= 6).